src/main/java/org/beta/tchap/identite/bot/BotSender.java

```diff
@@ -41,4 +47,25 @@ public boolean sendMessage(String serviceName, UserModel user, String friendlyCo
             }
         return true;
     }
+
+        /**
+     * Ensure there is a room where user is invited, if room existed, invitation is sent
+     * @param destMatrixId
+     * @return roomId or null if no existing room
+     */
+    private String ensureUserIsInRoom(String destMatrixId) {
+        DirectRoomsResource allRooms = this.matrixService.getRoomService().listBotDMRooms();
```

Comment:
commentaire non bloquant pour merger la PR : 
`this.matrixService.getRoomService()` pas tres fan de cet usage, ce serait equivalent a avoir `this.roomService` dans cette classe.
Peut-etre que la classe MatrixService n'a plus grande utilite a aggreger tous les sous services, car a l'usage on prefererait utiliser les sous services